plugins/available/nvm.plugin.bash

```diff
@@ -9,10 +9,18 @@ cite about-plugin
 about-plugin 'node version manager configuration'
 
 export NVM_DIR="${NVM_DIR:-$HOME/.nvm}"
+
+# first check if NVM is managed by brew
+NVM_BREW_PREFIX=""
+if _bash_it_homebrew_check
+then
+  NVM_BREW_PREFIX=$(brew --prefix nvm 2>/dev/null)
```

Comment:
In my (now old) message, I mentioned:
```bash
$(brew --prefix --installed $forumula)
```

With the `--installed` option, it only returns a value **if** the formula is installed.

Without it, when the formula is not installed, it would return where the formula _would be_ installed (vs returning nothing)

Maybe you can test this logic locally to confirm, but I think we want the `--installed` flag here.

Reply:
`brew --prefix --installed $formula` runs about 30x slower (over a second on my machine) than `brew --prefix $formula` (30ms) because it has to exec a ruby script instead of just staying in the `bash` portion of `brew`. And since we're checking the existence of the script just below, I don't think we should care about the specifics of `brew --prefix --installed nvm` vs just `brew --prefix nvm`